core/components/organisms/verticalNav/MenuItem.tsx

```diff
@@ -23,34 +24,25 @@ interface MenuPillsProps {
 
 interface MenuLabelProps {
   label: string;
-  disabled?: boolean;
-  isActive: boolean;
+  labelColor: TextColor;
 }
 
 interface MenuIconProps {
   isChildrenVisible?: boolean;
-  isActive?: boolean;
-  disabled?: boolean;
 }
 
 const MenuLabel = (props: MenuLabelProps) => {
-  const { label, disabled, isActive } = props;
+  const { label, labelColor } = props;
```

Comment:
Please add test cases for the new prop added.

Reply:
This prop passes the color to the label. So, I have added test case to check the color of the label in different states.